### PR TITLE
EnumFacetFilter UI changes

### PR DESCRIPTION
**Pull request by conundrumgirl**
https://sagebionetworks.jira.com/browse/PORTALS-1099 - Facet text should wrap correctly
https://sagebionetworks.jira.com/browse/PORTALS-1104 - Facet numbers should be right aligned
https://sagebionetworks.jira.com/browse/PORTALS-1105 - Facet show all should show total number of facets
https://sagebionetworks.jira.com/browse/PORTALS-1106
image

### CODE REVIEW

File: src/__tests__/lib/containers/widgets/EnumFacetFilter.test.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ beforeEach(() => init())
 describe('initialization', () => {
   it('should initiate selected items correctly', async () => {
     const checkboxes = container.querySelectorAll<HTMLInputElement>(
-      'input[type="checkbox"]',
+      'input[type="checkbox"]:not(#select_all)',
```
Comment (leem42) [inline src/__tests__/lib/containers/widgets/EnumFacetFilter.test.tsx]: neat!

File: src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx [hunk 2]
```diff
@@ -43,7 +44,8 @@ function valueToLabel(
   if (eh) {
     displayValue = eh ? eh.name : `unknown (${value})`
   }
-  return `${displayValue} (${count})`
+
+  return `${displayValue}`
```
Comment (leem42) [inline src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx]: no more count?

File: src/lib/utils/hooks/useGetEntityHeaders.ts [hunk 3]
```diff
@@ -17,7 +17,6 @@ export default function useGetEntityHeaders(props: UseGetProfilesProps) {
     const getData = async () => {
       // look at current list of data, see if incoming ids has new data,
```
Comment (leem42) [inline src/lib/utils/hooks/useGetEntityHeaders.ts]: this file is on deleted develop, do you need to run `git rm --cached  src/lib/utils/hooks/useGetEntityHeaders.ts`

File: src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx (hunk 2 above)
Comment (conundrumgirl) [reply src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx]: not here. We get it straight in html 
https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/422/files/08608266ddb63fe526422a151fb198777bf06d9e#diff-733576c09bfbc11b32f3d4c6a64d28e0R137

**Pull request merged**